src/data/LyricsGrabber.ts

```diff
@@ -34,15 +34,14 @@ export async function getLyrics(songTitle: string, artist: string) {
             throw new Error("Genius API: " + JSON.stringify(geniusResult));
         }
         const hits = geniusData?.response?.hits;
-        if (
-            hits?.filter(
```

Comment:
`filter` doesn't change the input array.
So in the case:
- `hits[0]` doesn't match the filter,
- `hits[1]` does match the filter;
the if condition will pass but `hits[0]` will be used instead of the correct `hits[1]`.